### DISCUSSION TITLE
屏東大學-新建無人機學科考場

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#153] [General]
**新聞內容:**
民航局近來公告證照2年到期需要換證皆需要再考學科，因此學科考試需求相當大。屏東縣目前僅有術科考場，缺乏學科考場，因此考生必須至外縣市考學科，增加交通上負擔。因此，屏東大學在屏商校區VAR中心成立無人機學科考場。
「無人機是一個新興的產業。」屏東大學資訊學院長王朱福指出，應用領域包含農藥噴灑、搜救勘災、VR環景拍攝導覽等，都需要結合資訊技術，屏東大學資訊學院在這些領域都有研發成果，未來也會開設相關課程來結合無人機證照，強化實務教學。
屏東大學AI數位轉型研究中心主任王隆仁強調，無人機未來可結合AI人工智慧、AIOT智能感應器、5G高速網路，應用上也可以協助企業轉型。
**新聞心得:**
在這之前我還不知道操作無人機還可以考取證照，未來也許無人機控制員學習更多高科技的應用後會是企業的一大助手！目前我只知道拍攝環境跟農藥方面，不知道原來還可以搜救！查了一下相關新聞發現在深山迷路或是火山爆炸的時候想找尋同伴，無人機都有派上用場欸！
**新聞來源:**
https://udn.com/news/story/6928/5985909

**Comment by cobb110202008** [comment]
在現在的時代，無人機的應用範圍十分廣泛，操作無人機的證照也是趨勢下的自然產物。

**Comment by briankuo123** [comment]
雖然無人機可以運用在很多方面，但是也是有很多地方開始禁止無人機的飛行了，也許是怕滿天的吾人機會太亂太吵吧。

**Comment by minniewei** [comment]
如果整天一大堆無人機在頭上飛來飛去心情也會很煩吧!!!但還是期待無人機在未來的應用!!

**Comment by wh20502** [comment]
我以為無人機跟遙控飛機一樣都不需要執照，突然長知識了，那跨年拍煙火的那群人真的很厲害。

**Comment by Yunhans** [comment]
雞腿可以換嗎

**Comment by LanningYang** [comment]
現在無人機的應用越來越多元
